examples/graphene_apps/python_worker/workloads.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Where is file being used?

Reply:
In "examples/graphene_apps/python_worker/avalon_worker/workload_processor.py "